.rubocop.yml

```diff
@@ -46,23 +48,42 @@ Style/DotPosition:
 Style/EmptyLineBetweenDefs:
   AllowAdjacentOneLineDefs: true
 
+# Maintain compatibility with older Rubies
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
 # Allow multiline block chains
 Style/MultilineBlockChain:
   Enabled: false
 
+# Indent one level for follow-up lines
+Style/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+# Indent one level for follow-up lines
+Style/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+# TODO: Discuss
+Style/MutableConstant:
+  Enabled: false
+
+# TODO: Discuss
+Style/ParallelAssignment:
+  Enabled: false
```

Comment:
Ah, this also was just moved, but I added the discuss bit to hopefully get some explanatory comment for it. I suppose I can just add `# @troesnerr always hated this rule` :laughing:.